include/seqan3/alignment/configuration/align_config_method.hpp

```diff
@@ -28,12 +29,53 @@ namespace seqan3::align_cfg
  *
  * \details
  *
- * The alignment algorithm can be categorised in different methods. For example, the
- * \ref seqan3::align_cfg::method_local "local" and the
- * \ref seqan3::align_cfg::method_global "global" alignment are two different methods, while the semi-global alignment
- * is a variation of the global alignment. This differentiation makes it possible to define a subset of configurations
- * that can work with a particular method. Since it is not possible to guess what the desired method for a user is,
- * there is no default and this configuration must always be provided for the alignment algorithm.
+ * There are several methods for sequence alignment. We distinguish between \ref seqan3::align_cfg::method_local "local"
+ * and \ref seqan3::align_cfg::method_global "global" alignments and the semi-global alignment in between, which is a
+ * variation of the global alignment.
+ *
+ * **Global Alignment**:\verbatim
+--T--CC-C-AGT--TATGT-CAGGGGACACG-A-GCATGCAGA-GAC
+  |  || |  ||  | | | |||    || | | |  | ||||   |
+AATTGCCGCC-GTCGT-T-TTCAG----CA-GTTATG-T-CAGAT--C\endverbatim
+ * Finding the optimal global alignment of two sequences is solved by the **Needleman-Wunsch algorithm** a “dynamic
+ * program”. Hence, the algorithm requires O(nm) time and memory.
```

Comment:
Is it important to mention that it is based on dynamic programing? If so, I would suggest:
```suggestion
Finding the optimal global alignment of two sequences is solved by the dynamic programing algorithm **Needleman-Wunsch**. Hence, the algorithm requires O(nm) time and memory.
```

Or we leave the information out completely, Needleman-Wunsch is so well known that you either know that already or can google it in a few seconds. ;)

Reply:
Also, we mention the time complexity here and a few sentences below again, I think it is enough to mention it only once.